serverless-stepf.yml

```diff
@@ -27,7 +29,8 @@ rBuilds:
             Next: Finished
       Finished:
         Type: Task
-        Resource: arn:aws:lambda:#{AWS::Region}:#{AWS::AccountId}:function:${self:service}-${self:provider.stage}-finished
+        Resource:
+          Fn::Sub: arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${self:provider.stage}-finished
```

Comment:
Is this going to work okay, with some things being substituted by serverless first, and some being substituted by CloudFormation?

Reply:
I don't know, I have that same question, haven't found an answer yet

Reply:
When I merge, it will deploy to staging which is fine, so I can find out quickly without breaking production

Reply:
🤞🏻

Reply:
<img width="776" alt="image" src="https://github.com/rstudio/r-builds/assets/266698/a9add1ca-447a-4116-bc67-85c4f894e01b">
🎉 the answer was yes!